test/unit/spec/RegistrationFormFactory_spec.js

```diff
@@ -130,5 +130,17 @@ function (SchemaProperty, RegistrationFormFactory) {
 
     });
 
+    describe('PasswordContainsUserName', function () {
```

Comment:
There are several more tests in registration_spec that test the different combinations of password and username. This test is mainly for null usecase.